### DISCUSSION TITLE
replace repository.url with new URL in all community packages

### DISCUSSION THREAD

**Discussion by MH4GF** [dotansimha/graphql-code-generator#8600] [General] [answered]
For packages that have been moved to the community repository, it may be a good idea to replace repository.url in package.json with the new URL.
I am ready to send out a PullRequest, but I mentioned it first to make sure everything is ok first: https://github.com/dotansimha/graphql-code-generator-community/pull/17

**Comment by charlypoly** [comment, marked as answer]
Hi @MH4GF,

Good catch! 🦅 
Feel free to change your PR to "ready to review"

**Reply by MH4GF** [reply]
Thanks for the reply 😄  I've changed it to ready.
